release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1198,6 +1209,8 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-release-note-scalability-and-performance-bug-fixes_{context}"]
 ==== Scalability and performance
 
```

Comment:
```[suggestion] 

```
Previously, entering an invalid value in any `cpuset` field under `spec.cpu` in the `PerformanceProfile` resource caused the webhook validation to crash.